src/content/docs/guides/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -1156,6 +1156,16 @@ On the JavaScript side, the `WebviewWindow` class is now exported in the `@tauri
 
 The `onMenuClicked` function was removed, you can intercept menu events when creating a menu in JavaScript instead.
 
+### Migrate Embedded External Binaries (Sidecar)
+
+In Tauri v1, the external binaries and their arguments were defined in the allowlist in the Tauri config. In v2, you use the new permissions system to do so instead. Read [Migrate Permissions](#migrate-permissions) for more information. Alternatively, if you are upgrading from v1, the `migrate` command in Tauri v2 CLI should handle this for you. Read [Automated Migration](#automated-migration) for more information.
```

Comment:
```suggestion
In Tauri v1, the external binaries and their arguments were defined in the allowlist. In v2, use the new permissions system. Read [Migrate Permissions](#migrate-permissions) for more information. 
```

I think there's no need to mention the migrate command, because this section assumes manual migration. It's more a reference so users can make the changes themselves.